### PR TITLE
Update feature-flags.md

### PR DESCRIPTION
**Pull request by mirie**
I'm suggesting that we add a section describing to folks how to add a feature flag. For full context, see the discussion in this thread - https://forem-team.slack.com/archives/C7GE84DEJ/p1660320626653809

### CODE REVIEW

**Review by andygeorge — APPROVED** [review]

File: docs/technical-overview/feature-flags.md [hunk 1]
```diff
@@ -79,6 +79,20 @@ To enable such a feature:
...
 
+Note that you made need to add a feature before enabling it.
+
+## Adding a feature flag
+
+To add a feature flag, you must be on the `/admin/feature_flags` page and click "Add Feature".
+
+![A screenshot of the "Add Feature" button](https://user-images.githubusercontent.com/16007075/184420699-4917228c-fad8-49ea-b68b-6b20db972e43.png)
+
+Next, input the name of the feature and follow the instructions on screen, then click "Add Feature".
```
Comment (Ridhwana) [inline docs/technical-overview/feature-flags.md:90]:
```suggestion
Next, input the name of the feature and then click "Add Feature".
```

File: docs/technical-overview/feature-flags.md [hunk 2]
```diff
@@ -79,6 +79,20 @@ To enable such a feature:
...
+
+![Screen Shot 2022-08-12 at 2 25 26 PM](https://user-images.githubusercontent.com/16007075/184421097-54d65918-cff1-4704-97ad-7ed408d988f1.png)
+
+Navigate back to `/admin/feature_flags` to enable your feature.
```
Comment (Ridhwana) [inline docs/technical-overview/feature-flags.md:94]: Configure your feature flag (if required) before clicking on "Fully enable".

A screenshot of the configuration screen for adding a feature flag

When you navigate back to `/admin/feature_flags`, you should see a green dot next to your newly added feature flag.

File: docs/technical-overview/feature-flags.md [hunk 3]
```diff
@@ -79,6 +79,20 @@ To enable such a feature:
...
+
+## Adding a feature flag
+
+To add a feature flag, you must be on the `/admin/feature_flags` page and click "Add Feature".
+
+![A screenshot of the "Add Feature" button](https://user-images.githubusercontent.com/16007075/184420699-4917228c-fad8-49ea-b68b-6b20db972e43.png)
+
+Next, input the name of the feature and follow the instructions on screen, then click "Add Feature".
+
+![Screen Shot 2022-08-12 at 2 25 26 PM](https://user-images.githubusercontent.com/16007075/184421097-54d65918-cff1-4704-97ad-7ed408d988f1.png)
```
Comment (Ridhwana) [inline docs/technical-overview/feature-flags.md:92]:
```suggestion
![A screenshot showing the screen to add a feature flag](https://user-images.githubusercontent.com/16007075/184421097-54d65918-cff1-4704-97ad-7ed408d988f1.png)
```

**Review by Ridhwana — COMMENTED** [review]
@michael-tharrington this looks great ✨ 

I updated the flow a little bit with some suggestions. I thought that this part particularly `Configure your feature flag (if required) before clicking on "Fully enable".` needed to be added.

**Comment by mirie** [comment]
Closing based on discussion here: https://forem-team.slack.com/archives/C7GE84DEJ/p1660581219044759?thread_ts=1660320626.653809&cid=C7GE84DEJ

**Pull request closed without merging**